.prettierrc.json

```diff
@@ -1,6 +1,14 @@
 {
-  "tabWidth": 4,
-  "printWidth": 120,
+  "semi": true,
+  "printWidth": 100,
   "singleQuote": true,
-  "semi": true
-}
\ No newline at end of file
+  "trailingComma": "all",
+  "overrides": [
+    {
+      "files": "*.d.ts",
+      "options": {
+        "trailingComma": "es5"
+      }
+    }
+  ]
+}
```

Comment:
this configuration could be a discussion form all team

Reply:
I will create a vote to review these points